pkg/collector/check/check_test.go

```diff
@@ -94,6 +95,42 @@ func TestDigest(t *testing.T) {
 	assert.Equal(t, 16, len(config.Digest()))
 }
 
+func TestAddMetrics(t *testing.T) {
+	metricsCfg, err := LoadCheck("foo", "tests/metrics.yaml")
```

Comment:
The standard folder name for test fixtures is `testdata`, could you please adjust that?